tests/unit-tests/AuthTests.swift

```diff
@@ -3,9 +3,11 @@
 //
 
 import XCTest
+import CryptoKit
```

Comment:
Is it for the jwt token generation for test?

Reply:
Yes.